ecommerce/extensions/payment/processors/braintree.py

```diff
@@ -103,48 +106,42 @@ def issue_credit(self, source, amount, currency):
                          transaction_id, result.message)
             raise GatewayError
 
-    def generate_client_token(self, user):
-        params = {}
-        customer = None
-        customer_id = "{}".format(user.id)
+    def log_deep_errors(self, errors):
+        for error in errors:
+            logger.error("[%s][%s] %s",
+                         error.attribute,
+                         error.code,
+                         error.message)
 
+    def get_or_create_customer(self, user):
+        customer_id = "{}".format(user.id)
```

Comment:
You don't need to use `str()` on anything using `format()`, it will do the casting by itself if the value is something as simple as an int.